default/scripting/species/common/happiness.macros

```diff
@@ -16,9 +16,9 @@ AVERAGE_HAPPINESS
             activation = And [
                 Planet
                 (Source.LastTurnConquered < CurrentTurn - 5)
```

Comment:
Should be not current turn not equal to LastTurnConquered I guess

Reply:
That would work too. Though with the new LastTurnColonized property we could probably just remove the check for LastTurnConquered entirely since I don't think they can happen on the same turn anyway.

Reply:
Yes, you are right